packages/avatar/__tests__/avatar.spec.ts

```diff
@@ -1,20 +1,11 @@
 import { mount } from '@vue/test-utils'
 import { nextTick } from 'vue'
 import Avatar from '../src/index.vue'
-import { IMAGE_SUCCESS, IMAGE_FAIL } from '@element-plus/test-utils'
-
-beforeAll(() => {
-  Object.defineProperty(global.Image.prototype, 'src', {
-    set(src) {
-      const event = new Event(
-        src === IMAGE_FAIL ? 'error' : 'load',
-      )
-      nextTick(() => this.dispatchEvent(event))
-    },
-  })
-})
+import { IMAGE_SUCCESS, IMAGE_FAIL, mockImageEvent } from '@element-plus/test-utils'
 
 describe('Avatar.vue', () => {
+  beforeAll(mockImageEvent)
```

Comment:
This might pollute other tests as it's a global setter, you need to restore it after use,  reference to `test-utils/define-getter`

Reply:
fixed